pkg/database/facade/filter.go

```diff
@@ -29,12 +28,8 @@ type DBFilter interface {
 // with the default selectors with non-specific DBMS functions
 type PartialDBFilterImpl struct{}
 
-func (PartialDBFilterImpl) KindFilter(cond sqlbuilder.Cond, kind string) string {
-	return cond.Equal("kind", kind)
-}
-
-func (PartialDBFilterImpl) ApiVersionFilter(cond sqlbuilder.Cond, apiVersion string) string {
-	return cond.Equal("api_version", apiVersion)
+func (PartialDBFilterImpl) KindApiVersionFilter(cond sqlbuilder.Cond, kind, apiVersion string) string {
+	return cond.And(cond.Equal("kind", kind), cond.Equal("apiVersion", apiVersion))
```

Comment:
Just after checking the tests...

```suggestion
	return cond.And(cond.Equal("kind", kind), cond.Equal("api_version", apiVersion))
```

I'm wondering why this is not being raised on the unit tests... any idea?

Reply:
Because the unit tests mock the database, so this is only raised when the actual table of the database don't have `apiversion` and have `api_version` instead as a column.

It's being raised in the integration test though.

BTW, now it's solved.